windows-container/windows-build/windows-go/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM microsoft/windowsservercore:1803
```

Comment:
there is an official golang image for 1803 https://github.com/docker-library/golang/blob/e6528749f50809c858c6b9bad86be8541de6368e/1.9/windows/windowsservercore-1803/Dockerfile

I also suspect we can use the linux version golang image for cross plat compile to windows binary. it will be much faster.